src/main/flight/navigation_rewrite.c

```diff
@@ -1611,11 +1611,18 @@ void calculateFarAwayTarget(t_fp_vector * farAwayPos, int32_t yaw, int32_t dista
  *-----------------------------------------------------------*/
 static uint32_t landingTimer;
 static bool hasHadSomeVelocity;
+static int32_t landingThrSum;
+static int32_t landingThrSamples;
 
 void resetLandingDetector(void)
 {
     landingTimer = micros();
     hasHadSomeVelocity = false;
+    
+    // When descent starts the throttle is at hover and quickly drops to gain descend velocity,
+    // Start with a fake low throttle average to avoid passing the test on that throttle drop.
+    landingThrSum = 8 * 1000;
+    landingThrSamples = 8;
```

Comment:
I agree it's a bit of a ugly hack (that happens to work beautiful). Making it independent of the 50hz update rate for landing timer would be good though.